browser/src/Editor/NeovimEditor/NeovimEditorActions.ts

```diff
@@ -52,8 +52,12 @@ export interface IAddBufferLayerAction {
 export interface ISetViewportAction {
     type: "SET_VIEWPORT"
     payload: {
-        width: number
-        height: number
+        width: number,
+        height: number,
+        focusedEditor?: {
```

Comment:
I'm wondering why we need separate `width` and `height` here - is there a way we can simplify/consolidate these? Ideally, `NeovimEditor` should only need to be concerned with its own width and height.

Reply:
(So in other words - it seems like we should only care about the width and height of the editor)